mysql-test/std_data/percona_bug_ps260_world200.sql

```diff
@@ -0,0 +1,246 @@
+INSERT INTO `city` VALUES (158,'Tungi','BGD','Dhaka',168702);
+INSERT INTO `city` VALUES (159,'Jessore','BGD','Khulna',139710);
+INSERT INTO `city` VALUES (160,'Comilla','BGD','Chittagong',135313);
+INSERT INTO `city` VALUES (161,'Nawabganj','BGD','Rajshahi',130577);
+INSERT INTO `city` VALUES (162,'Dinajpur','BGD','Rajshahi',127815);
+INSERT INTO `city` VALUES (163,'Bogra','BGD','Rajshahi',120170);
+INSERT INTO `city` VALUES (164,'Sylhet','BGD','Sylhet',117396);
+INSERT INTO `city` VALUES (165,'Brahmanbaria','BGD','Chittagong',109032);
+INSERT INTO `city` VALUES (166,'Tangail','BGD','Dhaka',106004);
+INSERT INTO `city` VALUES (167,'Jamalpur','BGD','Dhaka',103556);
+INSERT INTO `city` VALUES (168,'Pabna','BGD','Rajshahi',103277);
+INSERT INTO `city` VALUES (169,'Naogaon','BGD','Rajshahi',101266);
+INSERT INTO `city` VALUES (170,'Sirajganj','BGD','Rajshahi',99669);
+INSERT INTO `city` VALUES (171,'Narsinghdi','BGD','Dhaka',98342);
+INSERT INTO `city` VALUES (172,'Saidpur','BGD','Rajshahi',96777);
+INSERT INTO `city` VALUES (173,'Gazipur','BGD','Dhaka',96717);
+INSERT INTO `city` VALUES (174,'Bridgetown','BRB','St Michael',6070);
+INSERT INTO `city` VALUES (175,'Antwerpen','BEL','Antwerpen',446525);
+INSERT INTO `city` VALUES (176,'Gent','BEL','East Flanderi',224180);
+INSERT INTO `city` VALUES (177,'Charleroi','BEL','Hainaut',200827);
+INSERT INTO `city` VALUES (178,'Li�ge','BEL','Li�ge',185639);
+INSERT INTO `city` VALUES (179,'Bruxelles [Brussel]','BEL','Bryssel',133859);
+INSERT INTO `city` VALUES (180,'Brugge','BEL','West Flanderi',116246);
+INSERT INTO `city` VALUES (181,'Schaerbeek','BEL','Bryssel',105692);
+INSERT INTO `city` VALUES (182,'Namur','BEL','Namur',105419);
+INSERT INTO `city` VALUES (183,'Mons','BEL','Hainaut',90935);
+INSERT INTO `city` VALUES (184,'Belize City','BLZ','Belize City',55810);
+INSERT INTO `city` VALUES (185,'Belmopan','BLZ','Cayo',7105);
+INSERT INTO `city` VALUES (186,'Cotonou','BEN','Atlantique',536827);
+INSERT INTO `city` VALUES (187,'Porto-Novo','BEN','Ou�m�',194000);
+INSERT INTO `city` VALUES (188,'Djougou','BEN','Atacora',134099);
+INSERT INTO `city` VALUES (189,'Parakou','BEN','Borgou',103577);
+INSERT INTO `city` VALUES (190,'Saint George','BMU','Saint George�s',1800);
+INSERT INTO `city` VALUES (191,'Hamilton','BMU','Hamilton',1200);
+INSERT INTO `city` VALUES (192,'Thimphu','BTN','Thimphu',22000);
+INSERT INTO `city` VALUES (193,'Santa Cruz de la Sierra','BOL','Santa Cruz',935361);
+INSERT INTO `city` VALUES (194,'La Paz','BOL','La Paz',758141);
+INSERT INTO `city` VALUES (195,'El Alto','BOL','La Paz',534466);
+INSERT INTO `city` VALUES (196,'Cochabamba','BOL','Cochabamba',482800);
+INSERT INTO `city` VALUES (197,'Oruro','BOL','Oruro',223553);
+INSERT INTO `city` VALUES (198,'Sucre','BOL','Chuquisaca',178426);
+INSERT INTO `city` VALUES (199,'Potos�','BOL','Potos�',140642);
+INSERT INTO `city` VALUES (200,'Tarija','BOL','Tarija',125255);
```

Comment:
Is this the reduced set of INSERTs?

Reply:
It is. It only contains 200 cities, we need them up to 181 for the crash.